packages/components/src/mobile/image/image-editing-button.native.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { TouchableWithoutFeedback, View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { MediaEdit } from '../media-edit';
+import SvgIconCustomize from './icon-customize';
+import styles from './style.scss';
+
+const ImageEditingButton = ( {
+	isSelected,
+	isUploadFailed,
+	isUploadInProgress,
```

Comment:
Since we only use these props to decide _not_ to render the component, maybe this could be simplified by conditioning in the parent. Wdyt?

Reply:
Ohh I like that! Thanks for the suggestion, I'll update it =)